src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getWidth ($number, $cardWidth, $cardGorizMargin) {
+  @return (($number * $cardWidth) + ($cardGorizMargin * 3));
+}
```

Comment:
It's better to assign function parameters as variables, so as not to confuse them. And what is 3? it'better to use $cardGorizMargin * number - 1;